web/src/katex_server.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-console */
+import escape from "escape-html";
+import express from "express";
+import prometheus from "express-prom-bundle";
+import katex from "katex";
+
+const host = "localhost";
+const port = Number(process.argv[2] ?? "9700");
+if (!Number.isInteger(port)) {
+    throw new TypeError("Invalid port");
+}
+
+const app = express();
+app.use(
+    prometheus({
+        promClient: {collectDefaultMetrics: {}},
+        buckets: [0.0005, 0.001, 0.002, 0.005, 0.01, 0.02, 0.05, 0.1, 0.5],
+        httpDurationMetricName: "katex_http_request_duration_seconds",
+    }),
+);
+
+app.get("/", (req: express.Request, res: express.Response) => {
+    const content = req.query.content;
```

Comment:
Sticking this in the query string might make it possible to hit Node’s 8 KB maximum URL length, especially given that URL encoding expands some Unicode characters to 12 bytes. We probably want to use a POST body.